api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -86,4 +94,7 @@ message QuicProtocolOptions {
   // If not specified, no debug visitor will be attached to connections.
   // [#extension-category: envoy.quic.connection_debug_visitor]
   core.v3.TypedExtensionConfig connection_debug_visitor_config = 11;
+
+  // Configure a type of UDP cmsg to pass to listener filters via QuicReceivedPacket.
+  SaveCmsgConfig save_cmsg_config = 12;
```

Comment:
What will be the default behavior (i.e., if it is not set)?

Reply:
No cmsg will be saved to the QuicReceivedPacket. I've added a comment.